source/process/spec-process.rst

```diff
@@ -0,0 +1,46 @@
+Spec Review Process
+===================
+
+This is a rough guideline for the process for drafting and reviewing specs. It’s meant to be used as a starting point, but not all specs will follow this exact process - please use your judgment, as it’s something that will vary depending on the situation.
+
+1. Brainstorm / research.
+    - Research topic
```

Comment:
I don't think that's correct, at least in my local version it only bolds the point? Which is I think the formatting we wanted for lists?

But maybe @JeffSchering can comment on what the formatting should be, I'm 0/5. 

<img width="736" alt="screenshot 2017-01-23 15 28 52" src="https://cloud.githubusercontent.com/assets/9354841/22221321/7e3c193c-e180-11e6-9c20-4e2b56f464fe.png">